tests/api_tests/users/test_views.py

```diff
@@ -86,6 +88,7 @@ def setUp(self):
 
     def tearDown(self):
         super(TestUserDetail, self).tearDown()
+        Node.remove()
```

Comment:
Verify that we need these. If we do, this is definitely the right place for them.